packages/sit-onyx/src/i18n/locales/en-US.json

```
@@ -84,5 +86,21 @@
   "showTooltip": {
     "info": "Show info tooltip",
     "error": "Show error tooltip"
+  },
+  "colorScheme": {
+    "headline": "Change appearance",
+    "subtitle": "Please select the appearance of the application",
```

imho this sentence sounds a bit weird in that context. might discuss with jannick about rephrasing?
```suggestion
    "subtitle": "Here you can choose your preferred appearance of the application",
```

asking with "Please" sounds like there is a problem if you don't select something ¯\\\_(ツ)_\/¯